README.md

```
@@ -231,6 +231,11 @@ Another comment
 ---
 ```
 
+### Get Ticket Comments
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)